src/component/memberitem/index.tsx

```diff
@@ -18,7 +18,15 @@ function MemberItem(props: MemberItemProps) {
         user,
       }}
       iconButton={{
-        onClick: handleClick,
+        props: {
+          onClick: handleClick,
+        },
+        wrap(node) {
+          if (!handleClick) {
```

Comment:
Temporary workaround for not having a combination `isMember` and `isRequested` state which disables the invite deletion button.